src/coreclr/gc/gc.cpp

```diff
@@ -9472,13 +9543,15 @@ static int __cdecl cmp_mark_list_item (const void* vkey, const void* vdatum)
 }
 #endif // _DEBUG
 
+#ifdef USE_REGIONS
+uint8_t** gc_heap::get_region_mark_list (uint8_t* start, uint8_t* end, uint8_t*** mark_list_end_ptr)
+{
+    size_t lowest_region_number = (size_t)g_gc_lowest_address >> gc_heap::min_segment_size_shr;
+    size_t region_number = ((size_t)start >> gc_heap::min_segment_size_shr) - lowest_region_number;
```

Comment:
nit - would be nice to make a small helper get_region_index method for something like this - I'm sure we'll be using it in the future work.

Reply:
I implemented a helper get_basic_region_index_for_address - it computes the region_number as in the second line in the code you quoted, i.e. subtracts lowest_region_number.